### PR TITLE
Update Star Command

### LINKED ISSUE #441: [FEAT] - Modify `star` command
# Feature Request

## Is your feature request related to a problem? Please describe:

Currently the `star` command sends a DM to the target user. Unsolicited DMs are tactless and undesired.

## Describe the solution you'd like

Refactor the command to send the star embed in the channel where the command was called.

### PR DESCRIPTION
**Pull request by LandonLloyd**
# Pull Request

## Description:

Updates the Star command so that it sends a message in the channel not in the users DM's.
## Related Issue:

Closes #441 

## Scope:

Did you remember to update the `package.json` version number?

- [ ] Major Version Update: X.\_.\_ (for complete code refactors, or large PRs that adjust most of the codebase)
- [ ] Minor Version Update: \_.X.\_ (for the addition of new commands)
- [x] Patch Version Update: \_.\_.X (for bug fixes _in the code_.)
- [ ] No Version Update: \_.\_.\_ (no version update for additions to tests, documentation, or anything that isn't end-user facing.)

## Documentation:

For _any_ version updates, please verify if the [documentation page](https://beccalyria.nhcarrigan.com) needs an update. If it does, please [create an issue there](https://github.com/nhcarrigan/Becca-Lyria-documentation/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=update.md&title=%5BUPDATE%5D) to ensure it is not forgotten.

- [ ] My contribution does NOT require a documentation update.
- [x] My contribution DOES require a documentation update.

### CODE REVIEW

File: src/commands/general/star.ts [hunk 1]
```diff
@@ -76,9 +76,9 @@ const star: CommandInt = {
       attachment.push(new MessageAttachment("./img/star.png", "star.png"));
 
       // Send an embed message with the star to the user.
-      await userToStarMentioned.send(
+      await channel.send(
         new MessageEmbed()
-          .setTitle("You got a gold star!")
+          .setTitle(`Congragulations, ${userToStarMentioned.toString()}, you got a gold star!`)
```
Comment (nhcarrigan) [inline src/commands/general/star.ts:81]:
```suggestion
          .setTitle(`Congratulations, ${userToStarMentioned.toString()}, you got a gold star!`)
```

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
Quick typo fix.

**Comment by LandonLloyd** [comment]
Did those go through? Your review changes.

File: src/commands/general/star.ts [hunk 2]
```diff
@@ -76,9 +76,9 @@ const star: CommandInt = {
       attachment.push(new MessageAttachment("./img/star.png", "star.png"));
 
       // Send an embed message with the star to the user.
-      await userToStarMentioned.send(
+      await channel.send(
         new MessageEmbed()
-          .setTitle("You got a gold star!")
+          .setTitle(`Congratulations, ${userToStarMentioned.toString()}, you got a gold star!`)
```
Comment (nhcarrigan) [inline src/commands/general/star.ts:81]:
```suggestion
          .setTitle(
            `Congratulations, ${userToStarMentioned.username}, you got a gold star!`
          )
```
Tweak this to display the name correctly.

**Review by nhcarrigan — APPROVED** [review]
LGTM~! 🥳 

Thank you for your pull request! We look forward to additional contributions!

As an aside, I recommend reviewing our contributing docs for instructions on creating a new branch for your work, and how to bring your fork up to date with the original repo. 🙂

**Pull request merged**